00-intro.md

```diff
@@ -14,12 +14,12 @@ At a high level, computers do four things:
 -   run programs
 -   store data
 -   communicate with each other
--   interact with us
+-   and interact with us
 
 They can do the last of these in many different ways,
-including direct brain-computer links and speech interfaces.
-Since these are still in their infancy,
-most of us use windows, icons, mice, and pointers.
+including direct brain-computer and speech interfaces.
+Since these interfaces are still in their infancy,
+most of us use computer screens, mice, touchpads and keyboards.
```

Comment:
By getting rid of windows and pointers, we lose the reference to the "software" part of how computers interact with us (and vice-versa). In fact, maybe we can rephrase to explicitly differentiate between software and hardware interfaces?